### ISSUE TITLE
implicit package installation

### ISSUE THREAD

**Issue by ratnanil** [pampicloud/cma-week1#2]
https://github.com/pampicloud/cma-week1/blob/def2cf6735745bf0b540d558590c630858781f2f/assignment/assignment_2.Rmd#L44-L51

I see this function used very often in students submission and alway wonder where it comes from. I am not a big fan of the function, since an unwary user might install many packages without explicit content when running your code. I think installing packages should require explicit user consent.

**Comment by pampicloud** [comment]
Hi nils

This function only installs if the package is needed. So no packages are installed, if not needed.

Cheers
oli

**Comment by ratnanil** [comment]
Yes and that's my point. I want to decide myself if I want to install a missing package. Using `require()` on each library makes missing libraries very explicit.
